helm/aws-load-balancer-controller/values.yaml

```diff
@@ -184,6 +184,13 @@ env:
 # recommended if using the Amazon VPC CNI plugin.
 hostNetwork: false
 
+# Specifies the dnsPolicy that should be used for pods in the deployment
+#
+# This may need to be used to be changed given certain conditions. For instance, if one uses the cilium CNI
+# with certain settings, one may need to set `hostNetwork: true` and webhooks won't work unless `dnsPolicy`
+# is set to `ClusterFirstWithHostNet`. See https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy
+dnsPolicy: Default
```

Comment:
why not leave the default value as blank and let k8s use the appropriate default?

Reply:
Sounds fair. Let's try that. Otherwise, I was mistaken and we probably shouldn't use "Default" but instead It would actually be better to set it to what the API will use by default which is `ClusterFirst`. We'll go with blank for now and see how that goes.